src/cnaas_nms/api/device.py

```diff
@@ -127,6 +128,15 @@
                             example="RENEW")
 })
 
+stackmember_model = device_api.model('stackmember', {
+    'member_no': fields.Integer(required=False),
+    'hardware_id': fields.String(required=True),
+    'prioity_id': fields.Integer(required=False)
+})
+
+stackmembers_model = device_api.model('stackmembers', {
+    'stackmembers': fields.List(fields.Nested(stackmember_model)),
+})
```

Comment:
My editor complains about pep8 "two whitespace before defining class" after this

Reply:
Turned on auto removing of whitespaces now (should have done this months ago ..) and found whitespaces elsewhere but not there, so hopefully its somehow fixed

Reply:
Ah sorry I was unclear, my editor wants to have two newlines before defining a new class because of pep8, so it's too few newlines/whitespace not too many